### DISCUSSION TITLE
Overlay positioning based on an event or a point

### DISCUSSION THREAD

**Discussion by alirezamirian** [adobe/react-spectrum#7267] [General]
`useOverlayPosition` can be used to position an overlay based on another element (`targetRef`). I'm curious if there has been any internal discussion or plan for supporting positioning based on a pointer event or the coordinates of a given point. 
Use case being context menus.

**Comment by devongovett** [comment]
This is possible as of our latest release, which adds a getTargetRect prop to Popover. Here's an example that uses it: https://react-aria.adobe.com/Autocomplete#inline-completions
